### PR TITLE
1.0.0 Release PR

### PR DESCRIPTION
**Pull request by sprjr**
I've saved a release draft, and will follow up with a `CHANGELOG.md` update as well. At there any other changes we need to handle before cutting this and publishing?

### CODE REVIEW

**Comment by sprjr** [comment]
As an aside, I've updated from `0.0.9` in my app to the latest alpha1 version, and the migration was not super clean, although not very complicated. I think that may be the nature of having done several iterations and slices in different ways. I'll be glad once the churn has settled and I can go refactor things a bit to be more uniform across my redux store :)

If we do make changes like `...ListMeta.readXhrStatus` naming, etc, I'd love to have a migration guide to help make sure all the bases are covered when migrating. The worry I had here when upgrading was that `fooListMeta.oldProp === xhrStatuses.PENDING` would simply start failing. My fault too for not having render test coverage for those tho as well.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## [1.0.0](https://github.com/jmeas/redux-simple-resource/releases/tag/1.0.0-alpha1) (3/19/17)
+This shares the same release notes as `1.0.0-alpha1`, and is mostly used to
```
Comment (jmeas) [inline CHANGELOG.md]: new line between the header and content pls

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## [1.0.0](https://github.com/jmeas/redux-simple-resource/releases/tag/1.0.0-alpha1) (3/19/17)
```
Comment (jmeas) [inline CHANGELOG.md]: link needs to be `...tag/1.0.0`

File: CHANGELOG.md (hunk 1 above)
Comment (jmeas) [inline CHANGELOG.md]: I don't think we need to say "mostly" here. This sentence describes **all** that the release does, so we can say that.

**Comment by jmeas** [comment]
Squash into 1 commit with the message "1.0.0" pls

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,8 @@
...
+This shares the same release notes as `1.0.0-alpha1`, and is mostly used to
+formally suggest that we're done with pre-release API iteration & churn and
```
Comment (jmeas) [inline CHANGELOG.md]: I'd pick either the word "iteration" or "churn," but not both.

**Comment by jmeas** [comment]
Looks good to me! If you merge this in and run `npm publish`, it should be released 🎉 

Then you just need to cut the release on GH. The draft looks good so far: I'd just make the text the same as what's in the CHANGELOG in this PR.

Thanks @sprjr !

**Pull request merged**